daml-lf/encoder/src/test/scala/com/digitalasset/daml/lf/archive/testing/EncodeV1Spec.scala

```diff
@@ -143,44 +143,6 @@ class EncodeV1Spec extends AnyWordSpec with Matchers with TableDrivenPropertyChe
       val pkg1 = normalize(decodedPackage, hashCode, pkgId)
       pkg shouldBe pkg1
     }
-
-    "Encoding of function type with different versions should work as expected" in {
```

Comment:
Why don't we need this test anymore?

Reply:
Since #8080 we have any one way to encode function types.